File: backend/src/main/java/com/example/course_project_2023/service/dto/SearchedReviewDto.java
package com.example.course_project_2023.service.dto;

import java.util.Date;

public record SearchedReviewDto(
        Long id,
        String teacherName,
        String teacherSurname,
        Long teacherId,
        Short score,
        String nickname,
        String universityName,
        String subjectName,
        String reviewText,
        Date createdDate,
        Long likeCount,
        Long dislikeCount,
        String likeLink,
        String dislikeLink,
        Boolean isLiked,
        Boolean isDisliked
) {

}

File: frontend/src/app/models/SearchedReviewDto.ts
export interface SearchedReviewDto {
  id: number,
  teacherName:string,
  teacherSurname:string,
  teacherId: string,
  score: number,
  nickname:string,
  universityName:string,
  subjectName: string,
  reviewText: string,
  createdDate: Date,
  likeCount: number,
  dislikeCount: number,
  likeLink: string,
  dislikeLink: string,
  isLiked: boolean,
  isDisliked: boolean
}